src/Exception/InvalidStreamPointerPositionException.php

```diff
@@ -17,7 +17,7 @@ class InvalidStreamPointerPositionException extends RuntimeException implements
     public function __construct(
         string $message = 'Invalid pointer position',
         $code = 0,
-        Throwable $previous = null
+        ?Throwable $previous = null
```

Comment:
What's the point adding `?` if the default value is `null`?

It was fixed automatically or manually? If automatically I can see potential issues with code which implement interface/extend other class on which we do not have control.

Reply:
I did not make any changes manually. This is all phpcbf.

Reply:
@webimpress This makes sense to me. I was surprised the first time I saw that setting a default value of `null` on a typehinted argument meant that it was considered nullable; adding the explicit notation in the type declaration makes that abundantly clear to users. As such, I definitely support this sort of change.